ecommerce/constants.py

```diff
@@ -16,3 +16,29 @@
 ORDERED_VERSIONS_QSET_ATTR = "ordered_versions"
 
 BULK_ENROLLMENT_EMAIL_TAG = "bulk_enrollment"
+
+CYBERSOURCE_CARD_TYPES = {
+    "001": "Visa",
+    "002": "Mastercard",
+    "003": "American Express",
+    "004": "Discover",
+    "005": "Diners Club",
+    "006": "Carte Blanche",
+    "007": "JCB",
+    "014": "Enroute",
+    "021": "JAL",
+    "024": "Maestro (UK)",
+    "031": "Delta",
+    "033": "Visa Electron",
+    "034": "Dankort",
+    "036": "Carte Bancaires",
+    "037": "Carta Si",
+    "039": "EAN",
+    "040": "UATP",
+    "042": "Maestro (Intl)",
+    "050": "Hipercard",
+    "051": "Aura",
+    "054": "Elo",
+    "061": "RuPay",
+    "062": "China UnionPay",
+}
```

Comment:
Question: All these card codes are valid or just we need to be specific ?

Reply:
These are the card types supported by CyberSource, the keys are as detailed in the CS API documentation.